### PR TITLE
New types for redux-shortcuts 0.0.3

### PR DESCRIPTION
**Pull request by tstirrat**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. https://github.com/tstirrat/peshead/commit/8d31d7e61605450eca255b46bdde63d24c49f777 ([build result](https://travis-ci.org/tstirrat/peshead/builds/375746855?utm_source=github_status&utm_medium=notification))
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/redux-shortcuts/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Type definitions for redux-shortcuts 0.0.3
+// Project: https://github.com/nak2k/node-redux-shortcuts
+// Definitions by: Tim Stirrat <https://github.com/tstirrat>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import { ActionCreator, Action, Dispatch } from "redux";
+// import * as Mousetrap from 'mousetrap';
```
Comment (andy-ms) [inline types/redux-shortcuts/index.d.ts]: Why is this commented out?

File: types/redux-shortcuts/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+// Type definitions for redux-shortcuts 0.0.3
```
Comment (andy-ms) [inline types/redux-shortcuts/index.d.ts]: Just `0.0`, the patch version will be generated automatically.

File: types/redux-shortcuts/tslint.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (andy-ms) [inline types/redux-shortcuts/tslint.json]: This should be just `{ "extends": "dtslint/dt.json" }`.

**Review by ghost — CHANGES_REQUESTED** [review]

File: types/redux-shortcuts/index.d.ts (hunk 1 above)
Comment (tstirrat) [reply types/redux-shortcuts/index.d.ts]: I couldn't figure out how to re-export the `import * as Mousetrap`, nor the `MousetrapInstance`. Though I'll give it another shot

Comment (tstirrat) [reply types/redux-shortcuts/index.d.ts]: Thanks, that worked.

**Review by sheetalkamat — APPROVED** [review]

**Pull request merged**